client/lib/analytics/README.md

```diff
@@ -122,3 +139,16 @@ If we had instead used `calypso_add_cart_product` and `calypso_remove_cart_produ
 Finally, for consistency, the verb at the end should be in the form `add`, `remove`, `view`, `click`, etc and _not_ `adds`, `added`, `adding`, etc.
 
 With the exception of separating tokens with underscores, these rules do not apply to property names. `coupon_code` is perfectly fine.
+
+## Page Views Paths Conventions
```

Comment:
Maybe we can pull out the conventions section and move it above Usage. A lot of folks miss that tracks props need to be snake case in order to be recorded.